webapp/src/utils/test-utils.tsx

```diff
@@ -32,4 +32,21 @@ function replaceChaoticIds(container: HTMLElement) {
   return container
 }
 
-export { replaceChaoticIds }
+import React from 'react'
```

Comment:
To match our other files should the imports go to the top?

Reply:
updated!